src/ideascale/models/custom_fields.rs

```diff
@@ -21,7 +21,7 @@ impl Default for CustomFieldTags {
             proposal_solution: "problem_solution".to_string(),
             proposal_brief: "challenge_brief".to_string(),
             proposal_importance: "importance".to_string(),
-            proposal_goal: "describe_your_solution_to_the_problem".to_string(),
+            proposal_goal: "how_does_success_look_like_".to_string(),
```

Comment:
Again, this should bump on tests but we should have a ticket from whoever changed or decide to change this.

Reply:
Btw, this is part of configuration input for the command. It could be done externally if something changes, no need to change it code wise.

Reply:
Agree, but in code is much easier than in config file

Reply:
I still think that if we can update the default with the current value that is a nice UX improvement